package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "openmrs-owa-platformui",
+  "version": "0.1.0",
+  "description": "A very simple UI for the OpenMRS Platform to give access to the Addon Manager, User manager, and other tools.",
```

Comment:
User manager should also be capital M for manager

Reply:
PR had been updated